resources/lib/addon.py

```diff
@@ -215,11 +224,24 @@ def play(item_to_play):
 
 
 def clearcache():
-    shutil.rmtree(tempdir, True)
-    addon = xbmcaddon.Addon(id=__addonid__)
-    heading = addon.getLocalizedString(32004)
-    message = addon.getLocalizedString(32005)
-    xbmcgui.Dialog().notification(heading, message, xbmcgui.NOTIFICATION_INFO, 1000)
+    xbmc.log(f"Cache is being cleared", level=xbmc.LOGDEBUG)
+    try:
+        # delete the channels.xml
+        channel_file_path = os.path.join(tempdir, "channels.xml")
+        if os.path.isfile(channel_file_path):
+            os.remove(channel_file_path)
+        # delete all folders
+        for lsItem in os.listdir(tempdir):
+            pathToCheck = tempdir + "/" + lsItem
```

Comment:
```suggestion
            path_to_check = tempdir + "/" + ls_item
```

Python  normally uses `snake_case`

Reply:
![before-and-after-coding](https://user-images.githubusercontent.com/1620062/147837693-1853893e-ef16-4341-bc51-22f3ff150dcd.jpg)